superbench/benchmarks/micro_benchmarks/ib_validation_performance.py

```diff
@@ -299,7 +299,9 @@ def __prepare_general_ib_command_params(self):
         if self._args.gpu_dev is not None:
             gpu = GPU()
             if gpu.vendor == 'nvidia':
-                gpu_dev = f'--use_cuda={self._args.gpu_dev}'
+                # Perftest supports CUDA only in BW tests
+                if 'bw' in self._args.command:
```

Comment:
maybe move the check to line 299? --use_rocm is not available for latency test neither

Reply:
Good point, will update.